tests/role_moderation_test.py

```diff
@@ -24,8 +24,9 @@ def test_moderation(fc):
 
     with srunit.auth_db_session():
         sirepo.auth_db.UserRole.delete_all()
+        sirepo.auth_db.UserRoleInvite.delete_all()
     with pkunit.pkexcept('SRException.*moderationRequest'):
-        fc.sr_sim_data(sim_type='elegant', sim_name='SPEAR3')
+        fc.sr_sim_data(sim_type='myapp', sim_name='Scooby Doo')
```

Comment:
Now you can default these values:

```py
        fc.sr_sim_data()
```

[See sr_sim_data](https://github.com/radiasoft/sirepo/blob/c5bd99d398ea3f72e1221f057fca49e9b115f25e/sirepo/srunit.py#L541) to see the defaults.

Reply:
ad61fc813610afe4d4c57cbe0e4322c503d6ac6c